src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/RevocationResponder.cs

```diff
@@ -30,6 +30,9 @@ internal sealed class RevocationResponder : IDisposable
 
         public bool RespondEmpty { get; set; }
 
+        public TimeSpan RespondDelay { get; set; }
```

Comment:
```suggestion
        public TimeSpan ResponseDelay { get; set; }
```

Reads a bit better to me, particularly in the formatted strings.